System/mas.1d.sh

```diff
@@ -9,13 +9,19 @@
 # <bitbar.dependencies>mas</bitbar.dependencies>
 # <bitbar.abouturl>https://github.com/matryer/bitbar-plugins/blob/master/System/mas.1d.sh</bitbar.abouturl>
 
-UPDATES=$(/usr/local/bin/mas outdated);
+if test $UPDATE_COUNT -gt 0; then
```

Comment:
I think you jumped to far ahead here.  This doesn't test for `mas` this checks that somehow the variable `UPDATE_COUNT` is set and not 0.

`if [ -n "$(which mas)" ]` is probably closer to what you are looking for to see if `mas` is installed.